packages/braintree-integration/src/BraintreeAcceleratedCheckout/BraintreeAcceleratedCheckoutPaymentMethod.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { createCheckoutService, LanguageService } from '@bigcommerce/checkout-sdk';
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { PaymentFormService } from '@bigcommerce/checkout/payment-integration-api';
+
+import BraintreeAcceleratedCheckoutPaymentMethod from './BraintreeAcceleratedCheckoutPaymentMethod';
+
+describe('BraintreeAcceleratedCheckoutPaymentMethod', () => {
+    const checkoutService = createCheckoutService();
+    const checkoutState = checkoutService.getState();
+
+    const method = {
```

Comment:
I think would be better to create `getBraintreeAcceletatedCheckoutPaymentMethod` here `packages/test-utils/src/payment-methods.mock.ts`

What do you think?

Reply:
I don't think that this data will be useful for other places. So it might be better to keep it here.

Reply:
`packages/test-utils/src/payment-methods.mock.ts` is a global package for all integrations, therefore we can keep common test logic for all integration instead of adding provider specific data that used only in that package..

What do you think?

Reply:
I got it. Let's leave it as it is

Reply:
I think @bc-nick's idea is better to keep the payment method specific mocks in related packages. That way you don't need to have dependency on the test-utils package. Also ideally braintree mocks should only be used by braintree package in theory.